src/data/queries/linkTeaser.ts

```diff
@@ -30,17 +30,16 @@ export const data: QueryData<DataOpts, ParagraphLinkTeaser[]> = async (
   return entities
 }
 
-export const formatter: QueryFormatter<
-  ParagraphLinkTeaser[],
-  LinkTeaserType[]
-> = (entities: ParagraphLinkTeaser[]) => {
-  return entities.map((entity) => ({
+export const formatter: QueryFormatter<ParagraphLinkTeaser, LinkTeaserType> = (
```

Comment:
I reworked this one to only operate on one entity. In most cases, I would expect a data formatter to format a single entity.